Documentation/runtime-configuration.md

```diff
@@ -64,7 +64,19 @@ If you want to use the member API directly you can find the documentation [here]
 
 ### Update a Member
 
-If you would like to update a member IP address (peerURLs), first, we need to find the target member's ID. You can list all members with `etcdctl`:
+#### Update advertise client urls
+
+If you would like to update the advertise client urls of a member, you can simply restart
```

Comment:
My preference for instructions is always pronounless and imperative (_"to update the advertise client URLs of a member, simply restart that member..."_). But this probably needs a) a once-over on the whole doc (and other docs), and b) an ack/nack from our new benevolent doc overlord @joshix